charts/operatorhub/README.md

```diff
@@ -0,0 +1,73 @@
+
+# OperatorHub
+
+[OperatorHub](https://operatorhub.io/) is a central location to find the wide array of great Operators that have been built by the community. Packaging of the Operators indexed in OperatorHub.io relies on the  [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) (OLM) to install, manage and update Operators consistently on any Kubernetes cluster.
+
+The OperatorHub and OLM are [available](https://docs.openshift.com/container-platform/4.3/operators/olm-understanding-operatorhub.html) via the OpenShift Container Platform web console as well and is the interface that cluster administrators use to discover and install Operators.
+
+## Introduction
+
+This chart enables specific operators as it creates a subscription for the specific operator. For this, it creates a Subscription resource instance and a OperatorGroup for the relevant operator's subscription. A ClusterServiceVersion (CSV) is created automatically since the value of CSV is set in the Subscription, so all of the custom resources relevant to the operator are created.
+
+
+## Installing the chart
+
+To add the chart repo:
+
+```bash
+$ helm repo add rht-labs-charts https://rht-labs.github.io/charts
+```
```

Comment:
we changed the repo name so should probs be  `helm repo add rht-labs https://rht-labs.github.io/rht-labs`